app/controllers/payment-links/metadata/metadata-form.js

```diff
@@ -45,7 +64,7 @@ class MetadataForm {
     for (const fieldKey in fields) {
       const field = fields[fieldKey]
       field.validation.some((validationEntry) => {
-        const valid = validationEntry.validator(this.values[field.id])
+        const valid = validationEntry.validator(this.values[field.id], this.existingMetadata)
```

Comment:
Could not get it working with out adding the extra param `existingMetadata `

Reply:
Working now - removed the extra param.
Not sure why it was not working earlier.